docs/src/content/docs/ops/config/log.adoc

```diff
@@ -55,3 +55,29 @@ A multi-node persistent log implementation that uses a remote service to store t
 We currently offer the following remote log implementations, available in their own modules:
 
 * link:log/kafka[Kafka]: a log implementation that uses a Apache Kafka topic to store the log.
+
+[#epochs]
+== Epochs
+
+An *epoch* is a manually assigned, monotonically increasing integer used to identify the generation of the log in XTDB:
+
+* Epochs allow a cluster to safely reset its log state following partial log loss, corruption, or intentional recovery operations, without requiring full reindexing of storage data.
+* If not explicitly configured, nodes assume `epoch = 0`.
+
+[#epoch-configuration]
+=== Configuration
+
+To configure an epoch, specify the `epoch` field inside the node's log configuration:
+
+[source,yaml]
+----
+log: !<LogType>
+  epoch: <new-epoch>
+----
+
+Where:
+
+* `<LogType>` is the chosen log implementation (e.g., `!Kafka`, `!Local`).
+* `<new-epoch>` is a positive integer greater than the previous epoch.
+
+All nodes within the same cluster **must** use an identical epoch value at startup.
```

Comment:
could add the process here for bumping an epoch? i.e. shut everything down, update config, turn nodes back on again

Reply:
Unrelated suggestion: `All nodes within the same cluster` => `All nodes connected to the same object store` ('cluster' is a bit ambiguous)